code/script.js

```diff
@@ -1,9 +1,58 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+/* Hamgurger menu start */
+
+/* I select the Navbar and Ham elements using query selector */
+let navbar = document.querySelector(".navbar")
+let ham = document.querySelector(".ham")
+
+/* I added an Event Listener click to toggle the menu */
+ham.addEventListener("click", toggleHamburger)
+
+// toggles hamburger menu in and out when clicking on the hamburger
+function toggleHamburger(){
+  navbar.classList.toggle("showNav")
+  ham.classList.toggle("showClose")
 }
```

Comment:
Nice job! Think about giving all variables (including ids and classes) descriptive names. Maybe write out hamburger to avoid confusion, and the "showClose" - can you think of something more descriptive?: )